.github/workflows/build.yml

```diff
@@ -100,7 +100,7 @@ jobs:
 
       - name: Test
         run: |
-          mvn -B -e surefire:test -DargLine="-XX:MaxRAMPercentage=70.0" -Dsurefire.reports.directory=${GITHUB_WORKSPACE}/surefire-reports-aggregate
+          mvn -T3 -B -e surefire:test -DargLine="-XX:MaxRAMPercentage=90.0" -Dsurefire.reports.directory=${GITHUB_WORKSPACE}/surefire-reports-aggregate
```

Comment:
If we're going to be running 3 test VMs simultaneously, do we really want to set the max RAM percentage for each to be 90%?